coding/cpp_style_guide.rst

```diff
@@ -1984,432 +1982,64 @@ We should, of course, strive to minimize the amount of such information.
 
 .. _style-guide-cpp-6-1:
 
-6-1. Multiple statements per line SHOULD NOT be used.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code-block:: cpp
+6-1. We follow the clang-format Google style for code layout with a few exceptions.
```

Comment:
I personally like having the tool define the style, because then there can be no discussion. But I agree that making "We follow the Google style guide for code layout with a few exceptions" the rule (with a link) perhaps makes more sense here. And then keep the remainder of the section on clang-format intact but list the exceptions more explicitly and change the emphasis.